packages/calypso-products/src/get-price-tier-for-units.ts

```diff
@@ -1,8 +1,10 @@
 export interface PriceTierEntry {
 	minimum_units: number;
 	maximum_units?: undefined | null | number;
+	minimum_price: number;
 	minimum_price_display: string;
 	minimum_price_monthly_display: string;
+	maximum_price: number;
```

Comment:
I double-checked on the server and it appears that both of these are always set to integers.